pages/login.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
This is a copy of the component in sbc-common-components, except:
- updated image source to work in Nuxt
- cleaned up layout